tests/countries/test_cambodia.py

```
@@ -12,20 +12,14 @@
 
 from unittest import TestCase
 
-from holidays.countries.cambodia import Cambodia, KH, KHM
+from holidays.countries.cambodia import Cambodia
 from tests.common import CommonCountryTests
 
 
 class TestCambodia(CommonCountryTests, TestCase):
     @classmethod
     def setUpClass(cls):
```

Ruff (0.14.3)

Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)